replace tutorial sample to use as base deploy-image/v1-alpha as scaffold (#5992)
operator-framework/operator-sdk: website/content/en/docs/building-operators/golang/tutorial.md

When prompted, enter yes `y` for creating both the resource and controller.

```console

> $ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --plugins="deploy-image/v1-alpha" --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,modern,-v" --run-as-user="1001"

Reviewer: @NikhilSharmaWe we need to revert this one because in the tutorial we are giving the steps to implement the code.

However, wdyt about we can add a note here such as : 

**Note:** You will check that in this tutorial we will be providing all steps to let you know how to implement the project. However, as a follow up you might want to check that would be possible to have the whole code generated to deploy and manage an Operand(image) using the [Deploy Image plugin](add a link for the kubebuilder doc) with the command  `$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --plugins="deploy-image/v1-alpha" --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,modern,-v" --run-as-user="1001"`

This will scaffold the Memcached resource API at `api/v1alpha1/memcached_types.go` and the controller at `controllers/memcached_controller.go`.

> **Note:** You will check that in this tutorial we will be providing all steps to let you know how to implement the project. However, as a follow up you might want to check that would be possible to have the whole code generated to deploy and manage an Operand(image) using the [Deploy Image plugin][deploy-image-plugin-doc] with the command `$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --plugins="deploy-image/v1-alpha" --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,modern,-v" --run-as-user="1001"`

Reviewer: c/c @varshaprasad96

> **Note:** You will check that in this tutorial we will be providing all steps to let you know how to implement the project. However, as a follow up you might want to check that would be possible to have the whole code generated to deploy and manage an Operand(image) using the [Deploy Image plugin][deploy-image-plugin-doc] with the command `$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --plugins="deploy-image/v1-alpha" --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,modern,-v" --run-as-user="1001"`

Reviewer 2: Nit:
```suggestion
**Note:** In this tutorial we will be providing all the steps to show you how to implement an operator project. However, as a follow up you might want to check the [Deploy Image plugin][deploy-image-plugin-doc] with which it is possible to have the whole code generated to deploy and manage an Operand(image). To do so, you can use the the command `$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --plugins="deploy-image/v1-alpha" --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,modern,-v" --run-as-user="1001"`
```

Reviewer: done

The following steps will show how to deploy the operator on the cluster. However, to run locally for development purposes and outside of a cluster use the target `make install run`.

Note that by using this plugin the Operand image informed will be stored via an environment variable in the `config/manager/manager.yaml` manifest.

> Therefore, before run `make install run` you need to export any environment variable that you might have. Example:

Reviewer 2: ```suggestion
Therefore, before running `make install run` you need to export any environment variable that you might have. Example:
```

Reviewer: done
